Rakefile

```diff
@@ -63,6 +82,19 @@ namespace :smithy do
       raise 'Fixture models are out of sync.  Run bundle exec rake smithy:sync-fixtures to correct'
     end
   end
+
+  task 'rbs:unit' do
+    env = {
+      'SMITHY_RUBY_RBS_TEST' => 'true'
+    }
+    sh(env, 'bundle exec rake smithy:spec:unit')
+  end
+
+  task 'rbs:endpoints' do
+    task('smithy:spec:endpoints').invoke('rbs_test')
```

Comment:
Does the asme approach as spec:unit not work here? Setting of rbs env and calling spec endpoints? Since spec helper is called by the task, spec helper can be responsible for inserting. I think it should be the same speed?

Reply:
The issue is that we need to construct `RBS_TEST_OPTS` that include paths to every generated sig directory (which we only know inside the task) and `RBS_TEST_TARGET` (which we only know inside the task).  I couldn't find a clean way to break down that task other than adding an argument to it.